Source/Scene/Cesium3DTileset.js

```diff
@@ -630,6 +624,24 @@ define([
         },
 
         /**
+         * A 4x4 transformation matrix that transforms the tileset's root tile.
+         *
+         * @type {Matrix4}
+         * @default Matrix4.IDENTITY
+         */
+        modelMatrix : {
+            get : function() {
+                return this._modelMatrix;
+            },
+            set : function(value) {
+                this._modelMatrix = Matrix4.clone(value, this._modelMatrix);
+                if (defined(this._root)) {
+                    this._root.updateTransform();
+                }
```

Comment:
This is technically not needed. But is useful to have if the user wants to change the `modelMatrix` inside the ready promise before using the bounding volume.

```javascript
tileset.readyPromise.then(function(tileset) {
    tileset.modelMatrix = modelMatrix;
    var boundingSphere = tileset.boundingSphere;
    viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(0, -2.0, 0));
});
```